app/src/main/java/com/allan/boardbuddies/MainActivity.java

```diff
@@ -19,15 +29,39 @@ public class MainActivity extends AppCompatActivity {
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        notes.add(new Note("hello", "bob"));
-        notes.add(new Note("my", "bob"));
-        notes.add(new Note("friend", "bob"));
-        notes.add(new Note("Kaitlyn", "bob"));
 
-        recyclerView = findViewById(R.id.main_recycler_view);
-        recyclerView.setLayoutManager(new LinearLayoutManager(this));
-        NoteAdapter adapter = new NoteAdapter(notes);
-        recyclerView.setAdapter(adapter);
+        bottomNavigationView = findViewById(R.id.bottomNavigationView);
+        bottomNavigationView.setOnNavigationItemSelectedListener(this);
+        bottomNavigationView.setSelectedItemId(R.id.menu_notes);
+    }
+    NotesFragment notesFragment = new NotesFragment();
+    SharedFragment sharedFragment = new SharedFragment();
+    TrashFragment trashFragment = new TrashFragment();
+
+    @Override
+    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.menu_notes:
+                getSupportFragmentManager()
+                        .beginTransaction()
+                        .replace(R.id.frameLayout, notesFragment)
+                        .commit();
+                return true;
+
+            case R.id.menu_shared:
+                getSupportFragmentManager()
+                        .beginTransaction()
+                        .replace(R.id.frameLayout, sharedFragment)
+                        .commit();
+                return true;
 
+            case R.id.menu_trash:
+                getSupportFragmentManager()
+                        .beginTransaction()
+                        .replace(R.id.frameLayout, trashFragment)
+                        .commit();
+                return true;
```

Comment:
better to add a default case that returns false, and break everywhere else, then return true at the bottom